cmd/k8s-node-labeller/main.go

```diff
@@ -318,6 +351,13 @@ func generateLabels(lblProps map[string]*bool) map[string]string {
 			results[k] = v
 		}
 	}
+
+	// Add the new GPU labels
+	gpuLabels := generatePartitionLabels()
+	for k, v := range gpuLabels {
+		results[k] = v
+	}
```

Comment:
please follow the existing structure and just add a function for each of the labels you wanted to add to labelGeneators.